Makefile

```diff
@@ -315,7 +317,12 @@ test: unit-test e2e-test
 .PHONY: unit-test
 unit-test: envtest
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(TEST_SRC) -v  -coverprofile cover.out
+CLEANFILES += cover.out
 
 .PHONY: e2e-test
 e2e-test: ## Run e2e tests for the controller
 	go test ./tests/e2e/ -run ^TestOdhOperator -v --operator-namespace=${OPERATOR_NAMESPACE} ${E2E_TEST_FLAGS}
+
+clean:
+	chmod u+w -R $(LOCALBIN) # envtest makes its dir RO
+	rm -rf $(CLEANFILES)
```

Comment:
but in all, i wonder what is the requirement to remove all of these.
is that because the version of the tools in the "bin" need to be upgrade? 
or the odh-manifests is outdated?

Reply:
Because they are build artefacts, not part of the repository, and clean should remove them? @bartoszmajsak ?

Reply:
ok, now i read the original requirement from 696.

Reply:
I think it's actually both - as long as we don't have any other way to ensure that version we have in `bin` does not match what we want to fetch in Makefile I think `make clean` is a convenient target. 

> Because they are build artefacts, not part of the repository, and clean should remove them

In general `make clean` should more or less have a project folder in a similar state to fresh clone, so that falls into this category.